udata_front/theme/gouvfr/assets/js/components/Report/ReportModal.vue

```diff
@@ -74,7 +74,7 @@
                     :hasError="fieldHasError('message')"
                     :errorText="getErrorText('message')"
                     v-model="form.message"
-                    :placeholder="t('Reason of your report.\nDon\'t include any personal data.')"
```

Comment:
The issue was with the quote character right? not the newline?

Reply:
The issue was the use of `\` in the translation key. We can keep the `\n` on the value side.